README.md

```diff
@@ -2,12 +2,12 @@
 
 ### What is Stenography?
 
-Stenography is the fastest way to input text. Court reporters have used machine stenography since the early 1900s to take down spoken word in realtime.
+Stenography is the fastest way to type down text. Court reporters have used machine stenography since the early 1900s to create written records of oral discourse in realtime.
```

Comment:
Some stenographers (not me, personally) avoid equating typing to stenography. They actually *do* call it "writing" which is a little confusing within the context of the introduction.

> Stenography is the fastest way to write on a computer.

?

> create written records of human speech

could also work and may sound less scientific.

Reply:
I like both suggestions. The first one, in particular, qualifies "the fastest way", which was bothering me a bit. I'll make the changes right away.